client/gc_client.go

```diff
@@ -91,50 +89,6 @@ func (c *client) UpdateServiceSafePointV2(ctx context.Context, keyspaceID uint32
 	return resp.GetMinSafePoint(), nil
 }
 
-// WatchGCSafePointV2 watch gc safe point change.
-func (c *client) WatchGCSafePointV2(ctx context.Context, revision int64) (chan []*pdpb.SafePointEvent, error) {
```

Comment:
i think this can be kept? there is no v1 API for this

Reply:
Is there any cases that it's actually used?

Reply:
not by tidb. not sure if serverless use it.

if needed, i think we can remove it in other PR when you do refactor, this PR we can focus on redirect the other 2 API